cli.js

```diff
@@ -6,7 +6,7 @@ const argv = require('yargs')
     type: 'string'
   })
   .option('file', {
```

Comment:
the script only supports `README.md` or `renovate.json`, so I would add the .choices param (didn't test it out)

```js
  .option('file', {
    description: 'append empty space to README.md or update renovate.json (one needs a file diff to create a bump PR)',
    type: 'string',
    choices: ['README.md', 'renovate.json']
  })
```

Reply:
I was thinking that you can add a new line in any file (doesn't have to be `README.md`) but we can limit the script to just work with `README.md` or `renovate.json`. I will try with choices :)